_data/survey.csv

```diff
@@ -77,12 +77,12 @@ select_one / select_multiple,,,,,,
 ,label,Table of answers,✔,✔,,
 ,list-nolabel,"Goes with label, no label on answer just radios",✔,✔,,
 ,autocomplete,Autocomplete widget,✔,✔,search,Not supported on select_multiple in Enketo
-,likert,Shows question as likert widget,✔,✕,,
+,likert,Shows question as likert widget,✔,✔,,
 ,horizontal,Displays choices horizontally and automatically creates neat columns for multiple rows,✔,✔,,
 ,compact,Displays choices horizontally as compact as possible,✔,✔,horizontal-compact,
 ,compact-n,"Grid widget with max n columns where n is a number from 1 to 10, e.g. use compact-2 or compact-3",✔,✔,,
-,quick,Auto advance to next question upon selection,✔,✕,,
-,quickcompact,Displays choices horizontally as compact as possible. Auto advance to next question upon selection,✔,✕,quick compact,
+,quick,Auto advance to next question upon selection,✔,✔,,
+,quickcompact,Displays choices horizontally as compact as possible. Auto advance to next question upon selection,✔,✔,quick compact,

```

Comment:
I think these two were reversed (as Enketo doesn't support them). I'll add x-es to Enketo.